genomesubsampler/prototypeSelection.py

```diff
@@ -41,6 +41,56 @@
 import scipy as sp
 
 
+def _validate_parameters(dm, num_prototypes, seedset=[]):
+    '''Validate the paramters for each algorithm.
+
+    Parameters
+    ----------
+    dm: skbio.stats.distance.DistanceMatrix
+        Pairwise distances for all elements in the full set S.
+    num_prototypes: int
+        Number of prototypes to select for distance matrix.
+        Must be >= 2, since a single prototype is useless.
+        Must be smaller than the number of elements in the distance matrix,
+        otherwise no reduction is necessary.
+    seedset: iterable of str
+        A set of element IDs that are preferably selected for the resulting
```

Comment:
Based on your suggestion, I simplified the description as: "A set of element IDs that are pre-selected as prototypes. Remaining prototypes are then recruited with the prototype selection algorithm. Warning: It will most likely violate the global objective function."